core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/dot-experiment.utils.ts

```diff
@@ -1,16 +1,25 @@
+import { ChartData } from 'chart.js';
+import * as jStat from 'jstat';
```

Comment:
Why this way? can't you just do `import { jStat } from 'jstat'`?

Reply:
Let me try if I can import only method I used.

Reply:
Works.